app/views/spree/admin/pages/_form.html.erb

```diff
@@ -1,46 +1,56 @@
-<div data-hook="admin_page_form_fields">
-  <div class="left eight columns alpha" data-hook="admin_page_form_left">
+<div data-hook="admin_page_form_fields" class="row">
+  <div class="col-6">
     <%= f.field_container :title do %>
-      <%= f.label :title %> <span class="required">*</span><br />
-      <%= f.text_field :title, :class => 'fullwidth title' %>
+      <%= f.label :title, :class => "required" %>
```

Comment:
Can we please use the new ruby syntax for hashes here and in the rest of the views? `class: "required"`

Reply:
done https://github.com/solidusio-contrib/solidus_static_content/pull/28/files#diff-b1b936346cadca2fc15c7c6d89a9e358R4